config/environments/production.rb

```diff
@@ -24,7 +24,11 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
> we shouldn't because we don't have a js engine in production ... but I don't know if that is something required for puma to work...@miquel: is it?

That's correct for sass I'd say, since we don't include sass related stuff. For JS this should work, because there's no JS engine dependency. So, if the thing missing is stylesheet, then we'll get the "no sass gem could be found" (which is fine I'd say), and if it's JS then we'll hit a file in app/assets/javascripts.

@jordimassaguerpla  Either way we get the expected behavior.